config/jobs/kubernetes-sigs/special-resource-operator/special-resource-operator-presubmits.yaml

```
@@ -57,18 +21,6 @@ presubmits:
       description: "unit-tests the source code of special-resource-operator."
     spec:
       containers:
-      - image: golang:1.17
-        command:
-        - scripts/test-infra/unit_tests.sh
-  - name: pull-special-resource-operator-e2e-k8s
-    always_run: true
-    decorate: true
-    annotations:
-      testgrid-dashboards: sig-node-special-resource-operator
-      testgrid-tab-name: e2e-k8s
-      description: "e2e-k8s the source code of special-resource-operator."
-    spec:
-      containers:
-      - image: golang:1.17
+      - image: golang:1.18
         command:
-        - scripts/test-infra/e2e_k8s.sh
+        - ci/prow/unit_tests
```

If we want to match the `Makefile` target:
```suggestion
        - ci/prow/unit-test
```

---

This is not the script name, see https://github.com/qbarrand/oot-operator/blob/main/ci/prow/unit_tests.

IMO, the convention should be as follow:
* Prow refers to bash scripts in the `ci/prow/file_name` (example `ci/prow/unit_tests`)
* Those scripts can call a single Makefile target (such as `make unit-tests`) or can contains multiple instructions (such as e2e script will do)